comment_iowriter.go

```diff
@@ -6,17 +6,50 @@ import (
 	"strings"
 )
 
-var _ CommentService = &CommentWriter{}
+var _ CommentService = &RawCommentWriter{}
 
-type CommentWriter struct {
+// RawCommentWriter is comment writer which writes results to given writer
+// without any formatting.
+type RawCommentWriter struct {
 	w io.Writer
 }
 
-func NewCommentWriter(w io.Writer) *CommentWriter {
-	return &CommentWriter{w: w}
+func NewRawCommentWriter(w io.Writer) *RawCommentWriter {
+	return &RawCommentWriter{w: w}
 }
 
-func (s *CommentWriter) Post(c *Comment) error {
+func (s *RawCommentWriter) Post(c *Comment) error {
 	_, err := fmt.Fprintln(s.w, strings.Join(c.CheckResult.Lines, "\n"))
 	return err
 }
+
+var _ CommentService = &UnifiedCommentWriter{}
+
+// UnifiedCommentWriter is comment writer which writes results to given writer
+// in one of following unified formats.
+//
+// Format:
+//   - <file>: [<tool name>] <message>
+//   - <file>:<lnum>: [<tool name>] <message>
+//   - <file>:<lnum>:<col>: [<tool name>] <message>
+// where <message> can be multiple lines.
+type UnifiedCommentWriter struct {
+	w io.Writer
+}
+
+func NewUnifiedCommentWriter(w io.Writer) *UnifiedCommentWriter {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewUnifiedCommentWriter should have comment or be unexported